packages/core/contracts-ovm/insured-bridge/implementation/BridgePool.sol

```diff
@@ -21,10 +24,19 @@ import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
  * to post collateral by earning a fee per fulfilled deposit order.
  * @dev A "Deposit" is an order to send capital from L2 to L1, and a "Relay" is a fulfillment attempt of that order.
  */
-contract BridgePool is Testable {
+contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20 {
     using SafeERC20 for IERC20;
     using FixedPoint for FixedPoint.Unsigned;
 
+    // Token that this contract receives as LP deposits.
+    IERC20 public override l1Token;
```

Comment:
`BridgePool` now contains the `l1Token` that it is associated with. Previously the pool had no 1:1 mapping or direct relationship to this token (as no functionality that needed this was implemented yet).